ros_buildfarm/ci_job.py

```diff
@@ -303,7 +303,8 @@ def _get_ci_job_config(
 
         # only Ubuntu Focal has a new enough pytest version which generates
```

Comment:
this comment probably needs to be updated given that the logic for `pytest_junit_compliant` involves bionic.

Reply:
I don't see how the comment should change? The `pytest` version as of Focal is new enough and since we don't want to keep enumerating newer distros as the appear the condition checks for the opposite - not being Bionic - which is the only older Ubuntu distro used and older than Focal.

Reply:
the main point here that changes the whole thing is that somehow the code seems to affect only the following platforms: Bionic, Focal and new releases after Focal. I was not able to find where this context occur in the code.

Reply:
As far as I can tell the condition exactly implements what the comment describes. If there is no concrete proposal what to change I will go ahead and merge this as-is.